### ISSUE TITLE
KB-COMMENT-53: Section 9.7

### ISSUE THREAD

**Issue by ragges** [dfoxfranke/nts#140]
KB:

Is there anything to say about a target end-state where some peers can
successfully require NTS for all connections (i.e., no fallback of any
sort)?

> For the reasons described here, implementations SHOULD NOT revert
> from NTS-protected to unprotected NTP with any server without
> explicit user action.

Which would then force a choice between tolerating bad time and reusing
cookies, until user action can be obtained?

**Comment by dansarie** [comment]
Is there a functional difference between "no fallback without explicit user action" and "no fallback"?

**Comment by dsibold** [comment]
I have to confess that I don't understand the question

**Comment by dansarie** [comment]
Yes, the question is hard to parse. 

He seems to have misunderstood the draft somewhat. I think he's confusing cookie reuse with NTS stripping attacks, where an attacker is actively trying to cause a client to revert to unprotected NTP.

**Comment by ragges** [comment]
Proposed reply:
This is about not accepting a downgrade from NTS-protected to unprotected (traditional) NTP. Wether cookies are reused or not is another issue, and is only about unlinkability of the client vs resilience of the time link.
